CGPs/0001.md

```diff
@@ -0,0 +1,45 @@
+# CGP [0001]: Unfreeze Voter Rewards
+
+- Date: 2020-05-12
+- Author: @aslawson
+- Status: DRAFT
+- Governance Proposal ID #: [if submitted]
+- Date Executed: [if executed]
+
+## Overview
+
+This change unfreezes voter rewards by setting the target voting yield and adjustment factor in the EpochRewards smart contract. Currently these values are 0 so the voter rewards are 0.  Once the target is initialized and the adjustment factor added, the contract will adjust the voting yield based on the target voting gold fraction.
+
+Changes in parameters:
+targetVotingYieldParams.target: 0 --> 0.00016
+targetVotingYieldParams.adjustmentFactor:  0 --> 1 / 3650  (or 0.00027397260274)
+targetVotingYieldParams.max:  unchanged (0.0005)
+
+The means that voters will begin receiving rewards.  This provides incentive for users to maintain stake in the network and vote for validators they feel provide benefit to the network.
+
+## Proposed Changes
+
+1. Set Target Voting Yield
+  - Destination: EpochRewards, setTargetVotingYield
+  - Data: 160000000000000000000 (0.00016 * 10^24)
+  - Value: 0 (NA)
+2. Set Target Voting Adjustment Factor (and keep max the same)
+  - Destination: EpochRewards, setTargetVotingYieldParameters
+  - Data: max = 500000000000000000000 (0.0005 * 10^24), adjustmentFactor = 273972602740000000000 (0.00027397260274 * 10^24)
+  - Value: 0 (NA)
+
+## Verification
+
+- Methods called: 
```

Comment:
I think this section should have step-by-step "run these commands to verify this proposal does what I say it does" instructions

Reply:
If your plan was to add this after the proposal was submitted, please leave a comment indicating this

Reply:
I feel like it does -- it directs to the exact CeloCli commands to run and what to expect.  I could only make it more exact if I gave them a specific account voting account (eg my own)...

For checking the parameter value, you mean something like guiding them through using the truffle console to access the smart contract variables?